dependencies/lib/perl/Transcript.pm

```diff
@@ -68,6 +68,8 @@ sub get_feature {
 
 	return 'intergenic' if($bp > $self->end && $strand eq '+');
 	return 'intergenic' if($bp < $self->start && $strand eq '-');
+	return 'intergenic' if($bp < $self->start && $strand eq '+');#Tian
+        return 'intergenic' if($bp > $self->end && $strand eq '-');#Tian
```

Comment:
Note that the rest of the file uses a leading tab as indentation. This line uses spaces. Please change to a tab for consistency.